.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,219 @@
+name: Run tests
+
+on: [pull_request]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Setup PHP with PECL extension
+        uses: shivammathur/setup-php@v2
+        if: ${{ !env.ACT }}
+        with:
+          php-version: 8.1
+          tools: composer
+        env:
+          fail-fast: true
+
+      - name: Setup PHP with PECL extension locally
+        uses: shivammathur/setup-php@v2
+        if: ${{ env.ACT }}
+        with:
+          php-version: 8.0
```

Comment:
The tests should run on the same PHP version as the docker image used by the site; that way we can catch syntax issues and composer depend issues.

Reply:
I was wondering about this - the site is using 8.0 - https://github.com/UN-OCHA/common-design-site/blob/0332cb7a64ea3c27c1593125a99552b4c3d459a7/docker/Dockerfile#L7 - but is there a way to use an environment variable so we don't have to change it here when we change it in the dockerfile?

Reply:
We might be able to set an env var (using an action) based on the Dockerfile.